jmclient/jmclient/websocketserver.py

```diff
@@ -30,29 +31,30 @@ def connectionLost(self, reason):
         """
         WebSocketServerProtocol.connectionLost(self, reason)
         self.factory.unregister(self)
+        self.active_session = True
```

Comment:
Why is this here? I don't think we want the session to to be set 'active' when the connection is lost?

Reply:
I can't recall why I did that but it makes no sense indeed, reverting.